metadata-ingestion/src/datahub/ingestion/source/ldap.py

```diff
@@ -24,6 +24,28 @@
     CorpUserSnapshotClass,
 )
 
+# default mapping for attrs
+# general attrs
+attrs_mapping: Dict[str, Any] = {}
+attrs_mapping["urn"] = "sAMAccountName"
+
+# user related attrs
+attrs_mapping["fullName"] = "cn"
+attrs_mapping["lastName"] = "sn"
+attrs_mapping["firstName"] = "givenName"
+attrs_mapping["displayName"] = "displayName"
+attrs_mapping["manager"] = "manager"
+attrs_mapping["mail"] = "mail"
+attrs_mapping["departmentNumber"] = "departmentNumber"
+attrs_mapping["title"] = "title"
+
+# group related attrs
+attrs_mapping["group_urn"] = "cn"
+attrs_mapping["owner"] = "owner"
+attrs_mapping["managedBy"] = "managedBy"
+attrs_mapping["uniqueMember"] = "uniqueMember"
```

Comment:
What are these attributes? I'm not familiar with these or how they can be mapped to DataHub concepts.

DataHub groups have

- urn
- name
- members
- [optional] owners

Reply:
Should we just call uniqueMember as "member"? 

It's not clear what the difference between "member" and "uniqueMember" are - do you know?

Reply:
In my company we use "member" only, example:
member: CN=...
member: CN=...

Reply:
> What are these attributes? I'm not familiar with these or how they can be mapped to DataHub concepts.
> 
> DataHub groups have
> 
>     * urn
> 
>     * name
> 
>     * members
> 
>     * [optional] owners

Sorry, you are correct! I had somehow missed these. I have corrected this now.

Reply:
> Should we just call uniqueMember as "member"?
> 
> It's not clear what the difference between "member" and "uniqueMember" are - do you know?

I am calling this as `members` to keep this aligned with DH concepts. The existing behavior was to look for a `uniqueMember` attribute, so I am retaining that when the mapping is not provided for this one. 

@bda618 thanks for checking! we also just use `member` attribute.